apps/ledger-live-mobile/src/screens/CustomImage/PreviewPreEdit.tsx

```diff
@@ -327,7 +327,7 @@ const PreviewPreEdit = ({ navigation, route }: NavigationProps) => {
             <Button
               type="main"
               size="large"
-              outline={!isStaxEnabledImage}
+              outline
```

Comment:
sure about this?

Reply:
Yes, it's something I added during the PR but that got removed during pixel polish, on develop it's like that